great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -37,6 +40,9 @@ def __init__(self, runtime_environment: dict):
                 override both those set in config_variables.yml and the environment
         """
         self.runtime_environment = runtime_environment
+        # these attributes that are set downstream.
+        self._config_variables = None
+        self._project_config = None
```

Comment:
These are instantiated at the `AbstractDataContext` to keep the static analysis tool happy.  The attributes are set downstream. Should they be instantiated as `None` here?